apps/website/src/components/preview-page.tsx

```diff
@@ -326,16 +362,49 @@ export function PreviewPage(props: PreviewPageProps) {
 
   return (
     <>
-      <Head index={props.index} />
+      <Head>
+        <title>
+          {`${ACTION_VERB[type]} ${data.info.displayName} in Status`}
+        </title>
+
+        <meta property="og:url" content={`https://status.app${asPath}`} />
+        <meta
+          property="og:title"
+          content={`${ACTION_VERB[type]} ${data.info.displayName} in Status`}
+        />
+        {/* <meta
+          property="og:image"
+          content={`${urlOrigin}/assets/preview/entity.png`}
+          key="og:image"
+        /> */}
+        <meta property="og:description" content={data.info.description} />
+        <meta
+          property="twitter:title"
```

Comment:
>Twitter will read the og:title, og:image and og:description tags for their card, this is why next-seo omits twitter:title, twitter:image and twitter:description so not to duplicate.

twitter [docs](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/markup)

Reply:
I believe this is a  quote from https://github.com/garmeeh/next-seo and not from Twitter, regardless, the docs mention a fallback behavior. Originally sourced from https://metatags.io as per https://github.com/status-im/status-web/pull/411#discussion_r1212051994, will remove.

Fyi, the previews are not generated due to the Vercel's auth.

Reply:
removed in 92550e30